api/src/play_asset_pack_manager.gd

```diff
@@ -0,0 +1,54 @@
+# ##############################################################################
+#
+#	Copyright 2020 Google LLC
+#
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		https://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+#
+# ##############################################################################
+#
+# Singleton class that initializes the PlayAssetDelivery Android plugin and 
+# manages downloads of asset packs. Required to autoload this script by
```

Comment:
Discussed about this autoload design design with Jake on GVC. Made the decision to not require, but recommend autoload as the solution. Developers can also choose to instantiate and manage this Object on their own.